tests/unit/expected_config/config.conf

```
@@ -67,6 +67,10 @@ configuration:
     enable: true
     expireTime: 6s
     maxRetryTimes: 4
+  telemetry:
+    enabled: False
```

```suggestion
    enabled: false
```

I would presume, since it looks like YAML and the jinja template includes `| lower`.